experiments/weekly_tests/xilinx.yaml

```diff
@@ -0,0 +1,5 @@
+design_dirs:
+  - ooc/
```

Comment:
I bunch of the ooc designs will fail the xilinx flow because their ports cannot automatically be mapped to I/O pins. I think for the xilinx flow we should just do base/ designs and for xilinx_ooc we run just the ooc designs.